src/transformers/models/convbert/tokenization_convbert.py

```diff
@@ -44,6 +49,467 @@
     "YituTech/conv-bert-small": {"do_lower_case": True},
 }
 
+# Copied from transformers.models.bert.tokenization_bert.load_vocab
+def load_vocab(vocab_file):
+    """Loads a vocabulary file into a dictionary."""
+    vocab = collections.OrderedDict()
+    with open(vocab_file, "r", encoding="utf-8") as reader:
+        tokens = reader.readlines()
+    for index, token in enumerate(tokens):
+        token = token.rstrip("\n")
+        vocab[token] = index
+    return vocab
+
+# Copied from transformers.models.bert.tokenization_bert.whitespace_tokenize
+def whitespace_tokenize(text):
+    """Runs basic whitespace cleaning and splitting on a piece of text."""
+    text = text.strip()
+    if not text:
+        return []
+    tokens = text.split()
+    return tokens
+
+
+# Copied from transformers.models.bert.tokenization_bert.BasicTokenizer
+class BertTokenizer(PreTrainedTokenizer):
```

Comment:
This should become the `ConvBertTokenizer` class, not stay as a `BertTokenizer` :-)

Reply:
should i delete `class ConvBertTokenizer(BertTokenizer):` at line 515?